components/org.wso2.carbon.business.rules.web/src/components/common/error/InternalServerError.jsx

```diff
@@ -0,0 +1,45 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+import React from 'react';
+// Material UI Components
+import Typography from 'material-ui/Typography';
+import Paper from 'material-ui/Paper';
+// Styles
+import Styles from '../../../style/Styles';
+import '../../../index.css';
+
+/**
+ * Represents an Internal Server Error message display
+ */
+class InternalServerError extends React.Component {
+    render() {
+        return (
+            <Paper style={Styles.messageContainer}>
+                <Typography type="title">
```

Comment:
Use `'` instead of `"`. Apply for other places as well.

Reply:
ESLint ruleset allows double quotes when calling JSX components!

Reply:
k